src/main/java/com/epam/test_generator/config/TestListener.java

```diff
@@ -0,0 +1,18 @@
+package com.epam.test_generator.config;
+
+import java.util.Arrays;
+import org.springframework.context.ApplicationListener;
+import org.springframework.context.event.ContextRefreshedEvent;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TestListener implements ApplicationListener<ContextRefreshedEvent> {
+
+
+    @Override
+    public void onApplicationEvent(ContextRefreshedEvent contextRefreshedEvent) {
+        String[] beanDefinitionNames = contextRefreshedEvent.getApplicationContext()
+            .getBeanDefinitionNames();
+        System.out.println(Arrays.toString(beanDefinitionNames));
```

Comment:
Who to read sysout output? If logging is required, use logging framework.

Reply:
Yes, forgot to remove this.

Reply:
Done